astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit fedd7c52ba80e2246f83b8d1752baa3a153a33e2
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
@wafels: Apologies, I do not know what these astropy helpers mean ? Could you tell me. I did git commit . in a few and then was warned by @dpshelio to not do it, since then I haven't. And I won't from next time.